server/pom.xml

```diff
@@ -130,27 +131,16 @@
                                     <extraArgument>-XX:MaxMetaspaceSize=256m</extraArgument>
                                     <extraArgument>-XX:MaxDirectMemorySize=1024m</extraArgument>
                                     <extraArgument>-XX:-OmitStackTraceInFastThrow</extraArgument>
-                                    <extraArgument>-XX:+AggressiveOpts</extraArgument>
-                                    <extraArgument>-XX:+UseFastAccessorMethods</extraArgument>
                                     <extraArgument>-XX:-UseAdaptiveSizePolicy</extraArgument>
                                     <extraArgument>-XX:+HeapDumpOnOutOfMemoryError</extraArgument>
-                                    <extraArgument>-XX:HeapDumpPath=${BASEDIR}/logs/java_heapdump.hprof</extraArgument>
+                                    <extraArgument>-XX:HeapDumpPath=@BASEDIR@/logs/java_heapdump.hprof</extraArgument>
                                     <!--gc-->
                                     <extraArgument>-XX:+DisableExplicitGC</extraArgument>
-                                    <extraArgument>-XX:+UseParNewGC</extraArgument>
-                                    <extraArgument>-XX:+UseConcMarkSweepGC</extraArgument>
                                     <extraArgument>-XX:+CMSParallelRemarkEnabled</extraArgument>
-                                    <extraArgument>-XX:+UseCMSCompactAtFullCollection</extraArgument>
                                     <extraArgument>-XX:+UseCMSInitiatingOccupancyOnly</extraArgument>
                                     <extraArgument>-XX:CMSInitiatingOccupancyFraction=75</extraArgument>
-                                    <extraArgument>-Xloggc:${BASEDIR}/logs/seata_gc.log</extraArgument>
+                                    <extraArgument>-Xloggc:@BASEDIR@/logs/seata_gc.log</extraArgument>
                                     <extraArgument>-verbose:gc</extraArgument>
-                                    <extraArgument>-XX:+PrintGCDetails</extraArgument>
-                                    <extraArgument>-XX:+PrintGCDateStamps</extraArgument>
-                                    <extraArgument>-XX:+PrintGCTimeStamps</extraArgument>
```

Comment:
Why remove these?

Reply:
This three argument also removed or replaced in Java 11, please reference [Enable Logging with the JVM Unified Logging Framework-Convert GC Logging Flags to Xlog](https://docs.oracle.com/en/java/javase/11/tools/java.html#GUID-BE93ABDC-999C-4CB5-A88B-1994AAAC74D5)
![image](https://user-images.githubusercontent.com/13172836/67621098-da439100-f83f-11e9-9e5f-51cba03811f0.png)